### PR TITLE
Add ndb_utils.{get_multi,put_multi,delete_multi}.

### PR DESCRIPTION
**Pull request by oliverchang**
The NDB library currently doesn't allow more than 1000 entities for
put_multi, and 500 for put_multi/delete_multi.

### CODE REVIEW

**Comment by oliverchang** [comment]
TBR to fix exceptions.

File: src/python/datastore/ndb_utils.py [hunk 1]
```diff
@@ -13,7 +13,12 @@
 # limitations under the License.
 """NDB utilities. Provides utility functions for NDB."""
 
-DEFAULT_BATCH_SIZE = 1000
+from builtins import range
+
+from google.cloud import ndb
+
+_GET_BATCH_SIZE = 1000
+_PUT_BATCH_SIZE = 500
```
Comment (inferno-chromium) [inline src/python/datastore/ndb_utils.py]: nit: s/PUT/MODIFY ?

**Comment by oliverchang** [comment]
(Also filed https://github.com/googleapis/python-ndb/issues/318)

File: src/python/datastore/ndb_utils.py (hunk 1 above)
Comment (oliverchang) [reply src/python/datastore/ndb_utils.py]: done

**Pull request merged**